service/version_bundles.go

```diff
@@ -0,0 +1,78 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func newVersionBundles() []versionbundle.Bundle {
```

Comment:
I think to begin with we do not need version bundles in this operator. It does not implement the business logic the version bundles suggest. This is already done by the other operators. An operator only needs version bundles to guarantee versioning and communication with the end user. The cluster-operator is an internal detail we can more easily change however we like.

Reply:
As this is not needed, there's no HTTP interface in the cluster-operator (unless we want to provide healthz for k8s, but even still it would be easy to implement with stdlib) so I'm wondering if microkit is needed for anything? I grasp that operatorkit is partially build with microkit in mind, but it's not a hard dependency there - right? So what do you think? Is there a reason for microkit usage here and what would be the benefits?

Reply:
The core process is managed via microkit. It is the standard for running our daemons. The command line interface is the same for all services, except the flags it consumes at runtime. The whole flag and config file management is based on microkit. The microendpoints integrate directly with it and we want that for the health checks.